portfolio/src/main/webapp/index.html

```diff
@@ -8,15 +8,30 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <h4>About Me</h4>
-      <img src= "/images/profile.jpg" alt="Profile Picture" width=200px height=200px margin-right=15px>    
+      <h1>LORNA ARMSTRONG</h1>
```

Comment:
You can use the [CSS text-transform Property](https://www.w3schools.com/cssref/pr_text_text-transform.asp) in your CSS file
```
h1, h3, .banner {
  text-transform: uppercase;
}
```
The upside here is, it will ensure that you are being consistent (i.e. making all `h1`s and `h3`s uppercase, not leaving something by mistake) and you won't have to change all the texts if you change your mind and want to go back to normal `capitalize`.

Reply:
Thanks for the tip and for the reading material, Kazi!